rapcom/dispatcher.py

```diff
@@ -357,6 +363,9 @@ def _enable_logging(log_level):
     """
     if log_level not in (None, 'DEBUG', 'INFO', 'WARN', 'ERROR'):
         raise ValueError('Invalid log level "{}".'.format(log_level))
+    if '--log-level' in sys.argv:
+        sys.argv.remove('--log-level')
+        sys.argv.remove(log_level)
```

Comment:
You pass in values like `--log-level DEBUG`. 367 removes `--log-level`, 368 removes `DEBUG`.